pkgs/development/ocaml-modules/uuidm/default.nix

```diff
@@ -0,0 +1,28 @@
+{stdenv, fetchurl, ocaml, findlib}:
+
+stdenv.mkDerivation rec {
+  version = "0.9.5";
+  name = "uuidm-${version}"; 
+  src = fetchurl {
+    url = "http://erratique.ch/software/uuidm/releases/uuidm-${version}.tbz";
+    sha256 = "03bgxs119bphv9ggg97nsl5m61s43ixgby05hhggv16iadx9zndm";
+  };
+
+  unpackCmd = "tar -xf $curSrc";
+
+  buildInputs = [ocaml findlib];
+
+  configurePhase = "ocaml setup.ml -configure --destdir $prefix";
```

Comment:
Should be `--prefix` rather than `--destdir`: look at where `uuidtrip` gets installed.